[BUG] Failed to create CEF browser

OS: Linux Mint 22
Java 17 CN1 Project

Code:
```
if (BrowserComponent.isNativeBrowserSupported()) {
    Log.p("Native Browser Supported");
    try {
        if (browser == null) {
            browser = new BrowserComponent();
            browser.setPinchToZoomEnabled(true);
            browser.setNativeScrollingEnabled(false);
        }
        browser.setPage(buildMapHtml(place), null);
        add(BorderLayout.CENTER, browser);
        browserShown = true;
    } catch (RuntimeException ex) {
        Log.e(ex);
        browser = null;
    }
}
```

Log:
```
/bin/sh ./mvnw -Didea.version=2026.1.1 -Dmaven.ext.class.path=/home/francesco/intellijIDEA/plugins/maven/lib/intellij.maven.rt/maven-event-listener.jar -Djansi.passthrough=true -Dstyle.color=always -P idea-simulator -Dcodename1.platform=javase -DskipTests=true -Dmaven.repo.local=/home/francesco/.m2/repository verify -e
WARNING: package com.apple.eawt not in java.desktop
[INFO] Error stacktraces are turned on.
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO] 
[INFO] dailyroutine                                                       [pom]
[INFO] dailyroutine-cn1libs                                               [pom]
[INFO] NativeLogsReader                                                   [pom]
[INFO] dailyroutine-common                                                [jar]
[INFO] dailyroutine-javase                                                [jar]
[INFO] 
[INFO] ---------------< com.baeldung.cn1tutorial:dailyroutine >----------------
[INFO] Building dailyroutine 1.0-SNAPSHOT                                 [1/5]
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] 
[INFO] -----------< com.baeldung.cn1tutorial:dailyroutine-cn1libs >------------
[INFO] Building dailyroutine-cn1libs 1.0-SNAPSHOT                         [2/5]
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] 
[INFO] -------< com.baeldung.cn1tutorial:dailyroutine-NativeLogsReader >-------
[INFO] Building NativeLogsReader 1.0-SNAPSHOT                             [3/5]
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] 
[INFO] --- build-helper-maven-plugin:1.7:attach-artifact (attach-artifacts) @ dailyroutine-NativeLogsReader ---
[INFO] 
[INFO] ------------< com.baeldung.cn1tutorial:dailyroutine-common >------------
[INFO] Building dailyroutine-common 1.0-SNAPSHOT                          [4/5]
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- properties-maven-plugin:1.0.0:read-project-properties (default) @ dailyroutine-common ---
[INFO] 
[INFO] --- codenameone-maven-plugin:7.0.237:generate-gui-sources (generate-gui-sources) @ dailyroutine-common ---
Generating GUI sources
No GUI Entries available
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ dailyroutine-common ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 0 resource
[INFO] 
[INFO] --- maven-compiler-plugin:3.8.0:compile (default-compile) @ dailyroutine-common ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 32 source files to /home/francesco/CN1 Tutorial/dailyroutine/common/target/classes
[INFO] /home/francesco/CN1 Tutorial/dailyroutine/common/src/main/java/com/baeldung/cn1tutorial/DailyRoutine.java: Some input files use unchecked or unsafe operations.
[INFO] /home/francesco/CN1 Tutorial/dailyroutine/common/src/main/java/com/baeldung/cn1tutorial/DailyRoutine.java: Recompile with -Xlint:unchecked for details.
[INFO] 
[INFO] --- codenameone-maven-plugin:7.0.237:compliance-check (cn1-process-classes) @ dailyroutine-common ---
[INFO] Running bytecode compliance check against Codename One Java Runtime API
[INFO] See https://www.codenameone.com/javadoc/ for supported Classes and Methods
[INFO] Invocation rewrite summary: classes rewritten=0, callsites rewritten=0
[INFO] 
[INFO] --- codenameone-maven-plugin:7.0.237:css (cn1-process-classes) @ dailyroutine-common ---
[INFO] CSS sources unchanged since last compile.  Skipping CSS compilation
[INFO] 
[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ dailyroutine-common ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /home/francesco/CN1 Tutorial/dailyroutine/common/src/test/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.8.0:testCompile (default-testCompile) @ dailyroutine-common ---
[INFO] Nothing to compile - all classes are up to date
[INFO] 
[INFO] --- maven-surefire-plugin:3.0.0-M5:test (default-test) @ dailyroutine-common ---
[INFO] Tests are skipped.
[INFO] 
[INFO] --- codenameone-maven-plugin:7.0.237:attach-test-artifact (attach-test-artifact) @ dailyroutine-common ---
[INFO] 
[INFO] --- maven-jar-plugin:2.4:jar (default-jar) @ dailyroutine-common ---
[INFO] Building jar: /home/francesco/CN1 Tutorial/dailyroutine/common/target/dailyroutine-common-1.0-SNAPSHOT.jar
[INFO] 
[INFO] ------------< com.baeldung.cn1tutorial:dailyroutine-javase >------------
[INFO] Building dailyroutine-javase 1.0-SNAPSHOT                          [5/5]
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- codenameone-maven-plugin:7.0.237:prepare-simulator-classpath (prepare-simulator-environment) @ dailyroutine-javase ---
[INFO] Preparing Simulator Classpath
[INFO] 
[INFO] --- codenameone-maven-plugin:7.0.237:generate-javase-sources (add-se-sources) @ dailyroutine-javase ---
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ dailyroutine-javase ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /home/francesco/CN1 Tutorial/dailyroutine/javase/src/main/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.8.0:compile (default-compile) @ dailyroutine-javase ---
[INFO] Nothing to compile - all classes are up to date
[INFO] 
[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ dailyroutine-javase ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /home/francesco/CN1 Tutorial/dailyroutine/javase/src/test/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.8.0:testCompile (default-testCompile) @ dailyroutine-javase ---
[INFO] Nothing to compile - all classes are up to date
[INFO] 
[INFO] --- maven-surefire-plugin:3.0.0-M5:test (default-test) @ dailyroutine-javase ---
[INFO] Tests are skipped.
[INFO] 
[INFO] --- maven-jar-plugin:2.4:jar (default-jar) @ dailyroutine-javase ---
[INFO] 
[INFO] --- exec-maven-plugin:3.0.0:java (run-in-simulator-idea) @ dailyroutine-javase ---
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
Adding CEF to classpath
Retina Scale: 1.0
Adding watch folder /home/francesco/CN1 Tutorial/dailyroutine/common/src/main/java
Did not find hotswap-agent.properties file
SourceChangeWatcher running.  Watching directories [/home/francesco/CN1 Tutorial/dailyroutine/common/src/main/java]
Starting CSS Watcher for prefix 
Watching CSS files for changes: [/home/francesco/CN1 Tutorial/dailyroutine/javase/../common/src/main/css/theme.css]
Running CSS watch with args [/usr/lib/jvm/java-21-openjdk-amd64/bin/java, -jar, -Dcli=true, -Dcef.dir=/home/francesco/.m2/repository/com/codenameone/codenameone-cef/84.4.1-M3/codenameone-cef-84.4.1-M3-linux64.zip-extracted, -Dparent.port=46003, /home/francesco/.m2/repository/com/codenameone/codenameone-designer/7.0.237/codenameone-designer-7.0.237-jar-with-dependencies.jar-extracted/designer_1.jar, -css, -input, /home/francesco/CN1 Tutorial/dailyroutine/javase/../common/src/main/css/theme.css, -output, /home/francesco/CN1 Tutorial/dailyroutine/javase/../common/target/classes/theme.res, -merge, /home/francesco/CN1 Tutorial/dailyroutine/javase/../common/target/css/theme.css, -l, /home/francesco/CN1 Tutorial/dailyroutine/javase/../common/src/main/l10n, -watch]
CSS> Retina Scale: 1.0
CSS> Using stateless mode
CSS> Looks like this CSS file needs CEF on classpath... adding it now, and retrying...
CSS> Adding CEF to classpath /home/francesco/.m2/repository/com/codenameone/codenameone-cef/84.4.1-M3/codenameone-cef-84.4.1-M3-linux64.zip-extracted
CSS> SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
CSS> SLF4J: Defaulting to no-operation (NOP) logger implementation
CSS> SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
CSS> Retina Scale: 1.0
CSS> Using stateless mode
CSS> Updating merge file /home/francesco/CN1 Tutorial/dailyroutine/javase/../common/target/css/theme.css
CSS> Starting watch thread to watch [/home/francesco/CN1 Tutorial/dailyroutine/javase/../common/src/main/css/theme.css]
CSS> Compiling /home/francesco/CN1 Tutorial/dailyroutine/javase/../common/target/css/theme.css to /home/francesco/CN1 Tutorial/dailyroutine/javase/../common/target/classes/theme.res
CSS> File has not changed since last compile.
CSS> CSS file successfully compiled.  /home/francesco/CN1 Tutorial/dailyroutine/javase/../common/target/classes/theme.res
[EDT] 0:0:20,194 - Codename One revisions: 7dd4e7d08b3442d90959477ee52a5ae8c4361b29

[EDT] 0:0:20,194 - Native Browser Supported
[EDT] 0:0:20,209 - Attempting to create browser component
[AWT-EventQueue-0] 0:0:20,211 - Attempting to create browser component
[EDT] 0:0:20,234 - Exception: java.lang.UnsatisfiedLinkError - /home/francesco/.m2/repository/com/codenameone/codenameone-cef/84.4.1-M3/codenameone-cef-84.4.1-M3-linux64.zip-extracted/lib/linux64/libjcef.so: /home/francesco/.m2/repository/com/codenameone/codenameone-cef/84.4.1-M3/codenameone-cef-84.4.1-M3-linux64.zip-extracted/lib/linux64/libjawt.so: version `SUNWprivate_1.1' not found (required by /home/francesco/.m2/repository/com/codenameone/codenameone-cef/84.4.1-M3/codenameone-cef-84.4.1-M3-linux64.zip-extracted/lib/linux64/libjcef.so)
[EDT] 0:0:20,235 - Exception: java.lang.RuntimeException - Failed to create CEF browser
[EDT] 0:0:20,236 - Exception: java.lang.RuntimeException - Failed to create CEF browser
java.lang.UnsatisfiedLinkError: /home/francesco/.m2/repository/com/codenameone/codenameone-cef/84.4.1-M3/codenameone-cef-84.4.1-M3-linux64.zip-extracted/lib/linux64/libjcef.so: /home/francesco/.m2/repository/com/codenameone/codenameone-cef/84.4.1-M3/codenameone-cef-84.4.1-M3-linux64.zip-extracted/lib/linux64/libjawt.so: version `SUNWprivate_1.1' not found (required by /home/francesco/.m2/repository/com/codenameone/codenameone-cef/84.4.1-M3/codenameone-cef-84.4.1-M3-linux64.zip-extracted/lib/linux64/libjcef.so)
	at java.base/jdk.internal.loader.NativeLibraries.load(Native Method)
	at java.base/jdk.internal.loader.NativeLibraries$NativeLibraryImpl.open(NativeLibraries.java:331)
	at java.base/jdk.internal.loader.NativeLibraries.loadLibrary(NativeLibraries.java:197)
	at java.base/jdk.internal.loader.NativeLibraries.loadLibrary(NativeLibraries.java:139)
	at java.base/java.lang.ClassLoader.loadLibrary(ClassLoader.java:2444)
	at java.base/java.lang.Runtime.loadLibrary0(Runtime.java:916)
	at java.base/java.lang.System.loadLibrary(System.java:2063)
	at org.cef.SystemBootstrap$1.loadLibrary(SystemBootstrap.java:24)
	at org.cef.SystemBootstrap.loadLibrary(SystemBootstrap.java:36)
	at org.cef.CefApp.startup(CefApp.java:536)
	at com.codename1.impl.javase.cef.CEFBrowserComponent.create(CEFBrowserComponent.java:212)
	at com.codename1.impl.javase.cef.CEFBrowserComponent.create(CEFBrowserComponent.java:204)
	at com.codename1.impl.javase.cef.CEFBrowserComponent.create(CEFBrowserComponent.java:201)
	at com.codename1.impl.javase.cef.JavaCEFSEPort.createCEFBrowserComponent(JavaCEFSEPort.java:139)
	at com.codename1.impl.javase.cef.JavaCEFSEPort$2.run(JavaCEFSEPort.java:121)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:308)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:773)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:720)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:714)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
java.lang.RuntimeException: Failed to create CEF browser
	at com.codename1.impl.javase.cef.JavaCEFSEPort.createCEFBrowserComponent(JavaCEFSEPort.java:133)
	at com.codename1.impl.javase.cef.JavaCEFSEPort.createBrowserComponent(JavaCEFSEPort.java:98)
	at com.codename1.ui.BrowserComponent$1.run(BrowserComponent.java:156)
	at com.codename1.ui.Display.executeSerialCall(Display.java:1296)
	at com.codename1.ui.Display.processSerialCalls(Display.java:1280)
	at com.codename1.ui.Display.edtLoopImpl(Display.java:1223)
	at com.codename1.ui.Display.mainEDTLoop(Display.java:1076)
	at com.codename1.ui.RunnableWrapper.run(RunnableWrapper.java:139)
	at com.codename1.impl.CodenameOneThread.run(CodenameOneThread.java:179)
Caused by: java.lang.UnsatisfiedLinkError: /home/francesco/.m2/repository/com/codenameone/codenameone-cef/84.4.1-M3/codenameone-cef-84.4.1-M3-linux64.zip-extracted/lib/linux64/libjcef.so: /home/francesco/.m2/repository/com/codenameone/codenameone-cef/84.4.1-M3/codenameone-cef-84.4.1-M3-linux64.zip-extracted/lib/linux64/libjawt.so: version `SUNWprivate_1.1' not found (required by /home/francesco/.m2/repository/com/codenameone/codenameone-cef/84.4.1-M3/codenameone-cef-84.4.1-M3-linux64.zip-extracted/lib/linux64/libjcef.so)
	at java.base/jdk.internal.loader.NativeLibraries.load(Native Method)
	at java.base/jdk.internal.loader.NativeLibraries$NativeLibraryImpl.open(NativeLibraries.java:331)
	at java.base/jdk.internal.loader.NativeLibraries.loadLibrary(NativeLibraries.java:197)
	at java.base/jdk.internal.loader.NativeLibraries.loadLibrary(NativeLibraries.java:139)
	at java.base/java.lang.ClassLoader.loadLibrary(ClassLoader.java:2444)
	at java.base/java.lang.Runtime.loadLibrary0(Runtime.java:916)
	at java.base/java.lang.System.loadLibrary(System.java:2063)
	at org.cef.SystemBootstrap$1.loadLibrary(SystemBootstrap.java:24)
	at org.cef.SystemBootstrap.loadLibrary(SystemBootstrap.java:36)
	at org.cef.CefApp.startup(CefApp.java:536)
	at com.codename1.impl.javase.cef.CEFBrowserComponent.create(CEFBrowserComponent.java:212)
	at com.codename1.impl.javase.cef.CEFBrowserComponent.create(CEFBrowserComponent.java:204)
	at com.codename1.impl.javase.cef.CEFBrowserComponent.create(CEFBrowserComponent.java:201)
	at com.codename1.impl.javase.cef.JavaCEFSEPort.createCEFBrowserComponent(JavaCEFSEPort.java:139)
	at com.codename1.impl.javase.cef.JavaCEFSEPort$2.run(JavaCEFSEPort.java:121)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:308)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:773)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:720)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:714)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
java.lang.RuntimeException: Failed to create CEF browser
	at com.codename1.impl.javase.cef.JavaCEFSEPort.createCEFBrowserComponent(JavaCEFSEPort.java:133)
	at com.codename1.impl.javase.cef.JavaCEFSEPort.createBrowserComponent(JavaCEFSEPort.java:98)
	at com.codename1.ui.BrowserComponent$1.run(BrowserComponent.java:156)
	at com.codename1.ui.Display.executeSerialCall(Display.java:1296)
	at com.codename1.ui.Display.processSerialCalls(Display.java:1280)
	at com.codename1.ui.Display.edtLoopImpl(Display.java:1223)
	at com.codename1.ui.Display.mainEDTLoop(Display.java:1076)
	at com.codename1.ui.RunnableWrapper.run(RunnableWrapper.java:139)
	at com.codename1.impl.CodenameOneThread.run(CodenameOneThread.java:179)
Caused by: java.lang.UnsatisfiedLinkError: /home/francesco/.m2/repository/com/codenameone/codenameone-cef/84.4.1-M3/codenameone-cef-84.4.1-M3-linux64.zip-extracted/lib/linux64/libjcef.so: /home/francesco/.m2/repository/com/codenameone/codenameone-cef/84.4.1-M3/codenameone-cef-84.4.1-M3-linux64.zip-extracted/lib/linux64/libjawt.so: version `SUNWprivate_1.1' not found (required by /home/francesco/.m2/repository/com/codenameone/codenameone-cef/84.4.1-M3/codenameone-cef-84.4.1-M3-linux64.zip-extracted/lib/linux64/libjcef.so)
	at java.base/jdk.internal.loader.NativeLibraries.load(Native Method)
	at java.base/jdk.internal.loader.NativeLibraries$NativeLibraryImpl.open(NativeLibraries.java:331)
	at java.base/jdk.internal.loader.NativeLibraries.loadLibrary(NativeLibraries.java:197)
	at java.base/jdk.internal.loader.NativeLibraries.loadLibrary(NativeLibraries.java:139)
	at java.base/java.lang.ClassLoader.loadLibrary(ClassLoader.java:2444)
	at java.base/java.lang.Runtime.loadLibrary0(Runtime.java:916)
	at java.base/java.lang.System.loadLibrary(System.java:2063)
	at org.cef.SystemBootstrap$1.loadLibrary(SystemBootstrap.java:24)
	at org.cef.SystemBootstrap.loadLibrary(SystemBootstrap.java:36)
	at org.cef.CefApp.startup(CefApp.java:536)
	at com.codename1.impl.javase.cef.CEFBrowserComponent.create(CEFBrowserComponent.java:212)
	at com.codename1.impl.javase.cef.CEFBrowserComponent.create(CEFBrowserComponent.java:204)
	at com.codename1.impl.javase.cef.CEFBrowserComponent.create(CEFBrowserComponent.java:201)
	at com.codename1.impl.javase.cef.JavaCEFSEPort.createCEFBrowserComponent(JavaCEFSEPort.java:139)
	at com.codename1.impl.javase.cef.JavaCEFSEPort$2.run(JavaCEFSEPort.java:121)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:308)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:773)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:720)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:714)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

--- a/docs/developer-guide/The-Components-Of-Codename-One.asciidoc
+++ b/docs/developer-guide/The-Components-Of-Codename-One.asciidoc
@@ -2900,6 +2900,35 @@ wb.setURL("jar:///Page.html");
 
 IMPORTANT: On Android a native indicator might show up when the web page is loading. This can be disabled using the `Display.getInstance().setProperty("WebLoadingHidden", "true");` call. You need to invoke this once.
 
+==== Troubleshooting "Failed to create CEF browser" on Linux
+
+In the Simulator, `BrowserComponent` is rendered by the bundled https://bitbucket.org/chromiumembedded/java-cef[CEF] (Chromium Embedded Framework) port. On Linux, `BrowserComponent.isNativeBrowserSupported()` may return `true` and yet creating the component throws `RuntimeException: Failed to create CEF browser`, with a root cause similar to:
+
+[source]
+----
+java.lang.UnsatisfiedLinkError: .../codenameone-cef-*-linux64.zip-extracted/lib/linux64/libjcef.so:
+    .../lib/linux64/libjawt.so: version `SUNWprivate_1.1' not found
+    (required by .../lib/linux64/libjcef.so)
+----
+
+The native CEF library (`libjcef.so`) is loaded with a sibling `libjawt.so` shipped inside the CEF archive. That copy of `libjawt.so` expects symbols (`SUNWprivate_1.1`) provided by the JDK's own `libjawt.so` and the matching `libjvm.so`. When the JVM running the Simulator does not expose those symbols at load time -- typically because the dynamic linker does not have the JDK's native library directories on its search path -- the load fails before any browser is created. This is purely a Simulator/desktop concern; the same code runs unchanged on Android and iOS.
+
+The fix is to launch the IDE (and therefore the Simulator JVM) with `JAVA_HOME` and `LD_LIBRARY_PATH` pointing at a supported JDK. For example, on Linux Mint / Ubuntu wrap the IDE startup script:
+
+[source,bash]
+----
+#!/bin/bash
+export JAVA_HOME=/usr/lib/jvm/java-17-openjdk-amd64
+export LD_LIBRARY_PATH=$JAVA_HOME/lib:$JAVA_HOME/lib/server${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}
+exec /path/to/your/ide "$@"
+----
+
+After relaunching the IDE through this wrapper, `BrowserComponent` instantiation succeeds in the Simulator. Any JDK from 11 through 25 that is supported for running the Simulator works -- the important part is that `LD_LIBRARY_PATH` resolves to that JDK's `lib` and `lib/server` directories so the bundled `libjcef.so` finds a compatible `libjawt`/`libjvm` pair.
+
+TIP: If `BrowserComponent` instantiation fails on Linux, log the environment with `CN.getProperty("java.home", "<unset>")`, `CN.getProperty("JAVA_HOME", "<unset>")` and `CN.getProperty("LD_LIBRARY_PATH", "<unset>")`. An `<unset>` value for `LD_LIBRARY_PATH` while the IDE was launched from a desktop shortcut is the typical fingerprint of this problem -- desktop launchers do not source your shell profile.
+
+NOTE: This issue does not affect packaged native applications, only the desktop Simulator running on Linux.
+
 ==== BrowserComponent hierarchy
 
 When Codename One packages applications into native apps it hides a lot of details to make the process simpler. One of the things hidden is the fact that you aren't dealing with a JAR anymore, so

